website/content/docs/upgrading/instructions/upgrade-to-1-10-x.mdx

```diff
@@ -0,0 +1,222 @@
+---
+layout: docs
+page_title: Upgrading to 1.10.0
+description: >-
+  Specific versions of Consul may have additional information about the upgrade
+  process beyond the standard flow.
+---
+
+# Upgrading to 1.10.0
+
+<EnterpriseAlert />
+
+## Introduction
+
+This guide explains how to best upgrade a single Consul Enterprise datacenter to v1.10.0
+from a version of Consul that is forwards compatible with v1.10. If you are on a major 
+version of Consul prior to 1.8, you will need to complete and [upgrade to 1.8.13](/docs/upgrading/instructions) 
+or higher before continuing with this guide. If you are already on major versions 1.8 or 1.9
+this guide will go over the procedures required for upgrading to v1.10. This process
+will require intermediate version upgrades to a forwards compatible release of v1.8 or v1.9
+as well as other licensing related configuration changes. If you have multiple Consul datacenters
+then they are upgraded in the normal order with each upgrade taking into account the instructions below.
+
+For the open source version of Consul please follow the 
+[General Upgrade Process](/docs/upgrading/instructions/general-process).
+
+~> You can only upgrade to Consul Enterprise 1.10 from a version of Consul Enterprise 
+1.8 >= 1.8.13 or 1.9 >= 1.9.7. Other versions of Consul Enterprise are not forwards
+compatible with v1.10 and will cause issues during the upgrade that could result in
+agents failing to start due to [changes in the way we manage licenses](/docs/enterprise/license/faq).
+
+## Requirements
+
+- All Consul servers, clients and snapshot agents should be on a version of Consul >= 1.8.0 and < 1.10.0. If
+  they are not at the minimum version or higher, follow [normal upgrade procedures](/docs/upgrading) to get them there.
+
+## Assumptions
+
+This guides makes the following assumptions:
+
+- You are familiar with the [General Upgrade Process](/docs/upgrading/instructions/general-process).
+- You have the ability to run Consul CLI commands. 
+- If ACLs are in use then you possess a token with at least `operator:read` permissions.
+
+## Considerations
+
+The main breaking changes in Consul Enterprise 1.10 that need special handling during upgrading are
+the licensing changes outlined on the [Specific Version Details](/docs/upgrading/upgrade-specific#consul-1-10-0)
+page. There are notes on that page about other changes that might cause issues during an upgrade as well.
+You can find more granular details in the [licensing FAQ](/docs/enterprise/license/faq) as well as the full 
+[changelog](https://github.com/hashicorp/consul/blob/master/CHANGELOG.md#1100-june-22-2021).
+Looking through these changes prior to upgrading is highly recommended.
+
+## Procedures
+
+~> When any of the steps below call for upgrading components, we assume you follow the normal safe upgrade
```

Comment:
nit, suggested re-wording 
Any steps in the following section that mention upgrading servers/clients assume that normal safe upgrade procedures are followed. ..

Reply:
👍